stablehlo/dialect/ChloOps.td

```diff
@@ -673,6 +673,15 @@ def CHLO_ErfOp : CHLO_UnaryElementwiseOp<"erf",
   }];
 }
 
+def CHLO_ErfInvOp : CHLO_UnaryElementwiseOp<"erf_inv",
+    [HLO_CompatibleOperandsAndResultType], HLO_FpTensor, HLO_FpTensor> {
```

Comment:
Does this op support anything other than f32/f64? Should we verify type at all?

Reply:
In practice the lowering in mlir_hlo upcasts less wide types, but in this repo I don't think it makes sense to restrict it. Also, leaving it like this is consistent with the other CHLO ops